src/components/molecules/Header.jsx

```diff
@@ -1,31 +1,33 @@
-import styled from 'styled-components'
-import 'reset-css';
+import styled from "styled-components";
+import "reset-css";
 import { Link } from "react-router-dom";
 
 // import { baseTheme } from 'styles/theme'
 
 const StyledHeader = styled.header`
-    top:0;
-    right: 100px;
-    width: 100%;
-    height: 50px;
-    background: purple;
-    position: sticky;
-    color: white;
-    text-align: center;
-    margin-top: 5px;
```

Comment:
<img width="1144" alt="Screenshot 2023-03-19 at 09 32 39" src="https://user-images.githubusercontent.com/48246024/226155864-04cf7467-5148-43a7-bc4a-afb1a98687a0.png">

Reply:
i tought it would be more beautiful this way, idk why